common/src/commonMain/kotlin/io/xorum/codeforceswatcher/redux/ToastAction.kt

```diff
@@ -21,3 +22,11 @@ sealed class Message {
 fun String?.toMessage() =
     if (this.isNullOrEmpty()) Message.NoConnection
     else Message.Custom(this)
+
+fun Message.handle() =
```

Comment:
`handle()` is a bit misleading here because we have a CF `handle` as well. I suggest `toString()`.